data/mods/Magiclysm/Spells/monsterspells.json

```
@@ -129,11 +129,53 @@
     "min_aoe": 25,
     "max_aoe": 60,
     "aoe_increment": 5.0,
-    "spell_class": "KELVINIST",
     "base_casting_time": 100,
     "base_energy_cost": 150,
     "energy_source": "MANA",
-    "difficulty": 2,
     "damage_type": "fire"
+  },
+  {
+    "id": "krab_gaze",
+    "type": "SPELL",
+    "name": { "str": "Krabgek Gaze" },
+    "description": "Carcinization ray, go! (Krabgek version of Necrotic Gaze for balance purposes)",
```

```suggestion
    "description": "Carcinization ray, go!  (Krabgek version of Necrotic Gaze for balance purposes)",
```
Probably sees that as equivalent to two separate sentences, and description text tends to mandate double spaces between sentences for style reasons.

---

... that is very strange. I've never done double-spacing after a sentence before.